packages/aws-cdk-lib/aws-ec2/lib/vpc.ts

```diff
@@ -1429,6 +1519,24 @@ export class Vpc extends VpcBase {
       throw new Error('supply at most one of ipAddresses or cidr');
     }
 
+    // this property can be set to false if an IPv6_ONLY VPC is implemented in the future
+    this.useIpv4 = props.vpcProtocol === VpcProtocol.IPV4_ONLY || props.vpcProtocol === VpcProtocol.DUAL_STACK;
+    // use property to avoid lint errors
+    if (this.useIpv4) {
+      ; // pass
+    }
```

Comment:
Use it where it makes sense though, right? Where it allocates IPv4 resources? WHere it adds the IPv4 routes?